### DISCUSSION TITLE
MHI2 not entering dev menu after 'MOST not available' error

### DISCUSSION THREAD

**Discussion by dootzord** [Mr-MIBonk/M.I.B._More-Incredible-Bash#301] [General]
MHI2 not entering dev menu after 'MOST not available' error on attempted FW update from MHI2_ER_SKG11_P2104_1 to MHI2_ER_SKG11_K3343.

It is now seemingly stuck in radio and the buttons do not take me anywhere.

Please help.

**Comment by harman-f** [comment]
Did you cancel the update?
Do you have a picture of the install summary?

you either have to plug in a loop connector into the MOST part or change coding to remove most function. 

Unit is not in car, right?

**Reply by dootzord** [reply]
It stopped installing on this screen.
It was attempted connected to car yes.

![IMG_0022](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/140133330/1af07ac7-0ebc-4ec8-93fd-3c74c14622fb)

**Reply by harman-f** [reply]
Is unit rebooting all the time now?

M.ai.B was not installed on that unit?
if just stuck, try to install M.I.B via autorunswdl.txt

**Reply by dootzord** [reply]
It is booting fine, maybe a little slower. It just remains on the radio section and none of the hardware buttons work.
Can you link me to the autorunswdl.text technique? I can't seem to find it

**Reply by harman-f** [reply]
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/Prepare-M.I.B-SD-card-&-install-M.I.B#automatic-install-recommended

**Reply by dootzord** [reply]
Thankyou! Easy to do. Would you recommend I use M.I.B v3.7.1 or just 3.6?

**Reply by dootzord** [reply]
Ok, one more question. I have succesfully gone through the M.I.B auto-install. I am now stuck here. What should I do?

![IMG_0025](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/140133330/664eda5f-c693-458f-ac5c-37354a53d406)

**Reply by harman-f** [reply]
Use 3.7.1

None of the buttons are working?
Soft reboot…

**Reply by dootzord** [reply]
Yeah it has done a reboot and landed on this screen after going past the all-green 'magic' screen. 
Do I click 'back' or click on 'Launcher'?

**Reply by harman-f** [reply]
Back

**Reply by dootzord** [reply]
Seems to have restarted and installed but still no response from the buttons for me to get to any menus apart from radio screen here:

![IMG_0028](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/140133330/a74d3ffd-b8ec-4e74-94d9-c5166e9a651c)
![IMG_0026](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/140133330/1dc378bf-5af6-441d-ba5b-6cf1cd4ec652)

**Reply by harman-f** [reply]
Cancel Version back documentation


Now take FW and add autorunswdl.txt to install FW again.

**Reply by dootzord** [reply]
Will try that thankyou. Is there a way to work around SWOP error if it comes up again? Do I need to order a MOST loop cable?

**Comment by dootzord** [comment]
Wow, thanks so much. I made it in M.I.B. Still no audio yet but everything appears to function.
Can anyone advise me in disabling CP? (I assume that is why the volume is disabled) 

Am I in the right place here?

![IMG_0038](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/140133330/c9dec578-810e-4b13-a67b-c7081e6c2b24)

**Reply by harman-f** [reply]
Apply patch and sound should work, at least CP will be gone.

**Comment by dootzord** [comment]
I had to get a VCDS code removed for a canton issue, but now it’s ALL
WORKING. Thanks for making this possible

On Thu, 3 Aug 2023 at 3:38 am, harman-f ***@***.***> wrote:

> Apply patch and sound should work, at least CP will be gone.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/301#discussioncomment-6619446>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBNEHUTPYOAS53IRWNINMLTXTKGCNANCNFSM6AAAAAA2666KGM>
> .
> You are receiving this because you authored the thread.Message ID:
> <Mr-MIBonk/M.I.B.
> ***@***.***>
>
--

**Comment by dootzord** [comment]
Ok, a VCDS code (switching 5f module byte #4- to FF) for previous vehicle's Canton system was the only thing holding it back from outputting audio but it's now ALL WORKING. Thankyou so much for the help and working on this project!

Ps. I also updated my backup to the server since there wasn't one for this build.
